src/main/java/Orca.java

```diff
@@ -0,0 +1,166 @@
+import java.util.Scanner;
+
+public class Orca {
+    static final String LOGO = "     _______  ______    _______  _______ \n"
+            + "    |       ||    _ |  |       ||   _   | \n"
+            + "    |   _   ||   | ||  |       ||  |_|  | \n"
+            + "    |  | |  ||   |_||_ |       ||       | \n"
+            + "    |  |_|  ||    __  ||      _||       | \n"
+            + "    |       ||   |  | ||     |_ |   _   | \n"
+            + "    |_______||___|  |_||_______||__| |__| \n";
+    static final int MAX_TASKS = 100;
+
+    static Task[] tasks = new Task[MAX_TASKS];
+    static int taskCount = 0;
+    static Scanner in = new Scanner(System.in);
+    static String userInput = "";
+    static CommandType commandType;
+    static int taskNo;
+    static Task newTask;
+
+    public static void printGreetingMessage() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println(LOGO);
+        System.out.println("    Hello! I'm Orca, your assistant chatbot.");
+        System.out.println("    What can I do for you?");
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void printByeMessage() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println("    Bye. Hope to see you again soon!");
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void printUnknownCommandMessage() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println("    Sorry, I don't understand what you mean.");
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void printTasks() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println("    Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println("    " + (i + 1) + "." + tasks[i]);
+        }
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void printLatestAddedTask() {
+        System.out.println("    --------------------------------------------------");
+        System.out.println("    Got it. I've added this task:");
+        System.out.println("      " + tasks[taskCount - 1]);
+        System.out.println("    Now you have " + taskCount + " tasks in the list.");
+        System.out.println("    --------------------------------------------------\n");
+    }
+
+    public static void findCommandType() {
+        if (userInput.equals("bye")) {
```

Comment:
Perhaps this if/else if statement could be replaced by a switch/case statement?

Reply:
I used some `startswith()` so I don't think really I can do that.